### PR TITLE
Add SDK 2.0 Bigframes Tensorflow sample notebook

### PR DESCRIPTION
**Pull request by matthew29tang**
**REQUIRED:** Add a summary of your PR here, typically including why the change is needed and what was changed. Include any design alternatives for discussion purposes.

<br>
Official sample notebook for SDK 2.0 Bigframes Tensorflow Integration
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [ ] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [ ] Follow the style and grammar rules outlined in the above notebook template.
- [ ] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [ ] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/sdk/sdk2_bigframes_tensorflow.ipynb [hunk 1]
```diff
@@ -0,0 +1,653 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/official/sdk/sdk2_bigframes_tensorflow.ipynb:1]: <p>Cell needs a link back to the documentation at the end of the cell:</p><p> </p><p>Learn more about [...](...)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2256/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/sdk2_bigframes_tensorflow.ipynb:1]: <p>combine the 2 cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2256/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/sdk2_bigframes_tensorflow.ipynb:1]: <p>I don't think Torch... has anything to do with the code?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2256/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (andrewferlitsch) [inline notebooks/official/sdk/sdk2_bigframes_tensorflow.ipynb:1]: <p>from trained model. -&gt; from the trained model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2256/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Comment (matthew29tang) [reply notebooks/official/sdk/sdk2_bigframes_tensorflow.ipynb:1]: What should this link back to?

Comment (andrewferlitsch) [reply notebooks/official/sdk/sdk2_bigframes_tensorflow.ipynb:1]: To a web doc that describes Bigframes

**Comment by matthew29tang** [comment]
Thanks for the review, I've addressed all the comments now.

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**